.github/workflows/validation_trigger.yml

```diff
@@ -24,7 +24,8 @@ jobs:
     steps:
       - name: Trigger poliastro/validation
         run: |
-          curl -XPOST -u "${{ secrets.PAT_VALIDATION}}:${{secrets.PAT_VALIDATION_TOKEN}}" \
+          curl -v \
+          -XPOST -u "${{ secrets.VALIDATION_USER }}:${{ secrets.VALIDATION_PASS }}" \
```

Comment:
I think this syntax just doesn't work: https://docs.github.com/en/actions/reference/encrypted-secrets#example-using-bash let me try something different